python/examples/minimal/minimal.py

```diff
@@ -9,9 +9,24 @@
 
 import arflow
 
+import rerun as rr
```

Comment:
Should we use `rr` or `self.recorder`?

Reply:
Ah yeah we should use self.recorder; that was for testing a new API in the new rerun version which I've pushed a commit to remove the testing.